src/widget/rating/rating.js

```diff
@@ -0,0 +1,117 @@
+import RangeWidget from '../../widget/range/range-widget';
+import { isNumber } from '../../js/utils';
+
+/**
+ * @extends RangeWidget
+ */
+class RatingWidget extends RangeWidget {
```

Comment:
Are we using any of the functions in the RangeWidget? If not, we should extend Widget instead.

Reply:
I see you are using `get props()`. Makes sense.

Reply:
changed to extend Widget class directly by adding it's own `get props()`

Reply:
Nice!